packages/cli/generators/project/templates/README.md.ejs

```diff
@@ -1,3 +1,15 @@
 # <%= project.name %>
 
 [![LoopBack](https://github.com/strongloop/loopback-next/raw/master/docs/site/imgs/branding/Powered-by-LoopBack-Badge-(blue)-@2x.png)](http://loopback.io/)
+
+## Installation
+
+Describe the installation steps.
```

Comment:
I think the generator can pre-fill this section, the typical instructions is to `npm install` the package, right?